webpack-test/cases/loaders/context/test.filter.js

```diff
@@ -1,12 +0,0 @@
-
-/*
-const supportsRequireInModule = require("../../../helpers/supportsRequireInModule");
-
-module.exports = config => {
```

Comment:
In my opinion, this ﻿`module.exports` should not be deleted.

Reply:
I delete test.filter.js according to https://github.com/web-infra-dev/rspack/blob/77f57e52180462019274bada5a2b77e90a6ffca4/webpack-test/README.md?plain=1#L17-L24